beetsplug/web/__init__.py

```diff
@@ -69,10 +69,14 @@ def json_generator(items, root):
             first = False
         else:
             yield ','
-        yield json.dumps(_rep(item))
+        yield json.dumps(_rep(item, expand=expand))
     yield ']}'
 
 
+def is_expand(args):
```

Comment:
Hmm, we'd be better with @sampsyo's opinion on this I think.

Here's how I'd (personally) write the method:

```python
def is_expand():
    """Returns whether the current request is for an expanded response."""

    return flask.request.args.get('expand') is not None
```

We can directly access `flask.request.args` without passing it in as flask does some magic with the `flask.request` object per-request so that it is the current request.